kge/model/embedder/lookup_embedder.py

```diff
@@ -77,6 +78,48 @@ def normalize_embeddings(job):
 
             job.pre_batch_hooks.append(normalize_embeddings)
 
+    @torch.no_grad()
+    def init_pretrained(
+        self,
+        pretrained_embedder: torch.nn.Embedding,
+        ids_pretrained_model: List,
+        ids_new_model: List,
```

Comment:
Are these two parameters really needed? Both this and the pretrained embedder know their dataset, which in turn knows the IDs.